diff --git a/node_modules/socket.io-parser/build/cjs/index.js b/node_modules/socket.io-parser/build/cjs/index.js
index 8166a28..c29b7bf 100644
--- a/node_modules/socket.io-parser/build/cjs/index.js
+++ b/node_modules/socket.io-parser/build/cjs/index.js
@@ -118,7 +118,8 @@ class Decoder extends component_emitter_1.Emitter {
         let packet;
         if (typeof obj === "string") {
             if (this.reconstructor) {
-                throw new Error("got plaintext data when reconstructing a packet");
+                console.error('ABOUT TO THROW', obj);
+                throw new Error("### got plaintext data when reconstructing a packet");
             }
             packet = this.decodeString(obj);
             if (packet.type === PacketType.BINARY_EVENT ||
